### PR TITLE
[spark] Add sagemaker processing notebook example

### PR DESCRIPTION
**Pull request by xyang16**

### CODE REVIEW

File: apache-spark/notebook/sagemaker_spark_processing_text.ipynb [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+    "\n",
+    "input_bucket = \"alpha-djl-demos\"\n",
+    "input_prefix = \"dataset/nlp/amazon_reviews\"\n",
+    "\n",
+    "timestamp_prefix = strftime(\"%Y-%m-%d-%H-%M-%S\", gmtime())\n",
+    "prefix = \"sagemaker/spark-processing-djl-demo/{}\".format(timestamp_prefix)\n",
+    "output_bucket = sagemaker_session.default_bucket()\n",
+    "output_prefix = \"{}/output/processed\".format(prefix)\n",
+    "\n",
+    "image_uri = \"125045733377.dkr.ecr.us-east-1.amazonaws.com/djl-sagemaker-spark-processing:latest\"\n",
```
Comment (frankfliu) [inline apache-spark/notebook/sagemaker_spark_processing_text.ipynb:160]:
```suggestion
    "image_uri = \"125045733377.dkr.ecr.us-east-1.amazonaws.com/djl-sagemaker-spark-processing\"\n",
```

File: apache-spark/notebook/sagemaker_spark_processing_text.ipynb [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+   "source": [
+    "from sagemaker.spark.processing import PySparkProcessor\n",
+    "\n",
+    "input_bucket = \"alpha-djl-demos\"\n",
+    "input_prefix = \"dataset/nlp/amazon_reviews\"\n",
+    "\n",
+    "timestamp_prefix = strftime(\"%Y-%m-%d-%H-%M-%S\", gmtime())\n",
+    "prefix = \"sagemaker/spark-processing-djl-demo/{}\".format(timestamp_prefix)\n",
+    "output_bucket = sagemaker_session.default_bucket()\n",
+    "output_prefix = \"{}/output/processed\".format(prefix)\n",
```
Comment (frankfliu) [inline apache-spark/notebook/sagemaker_spark_processing_text.ipynb:158]:
```suggestion
    "output_prefix = \f"{prefix}/output/processed"\n",
```

File: apache-spark/notebook/sagemaker_spark_processing_text.ipynb [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+    "\n",
+    "    df = spark.read.json(\"s3://\" + os.path.join(args.s3_input_bucket, args.s3_input_key_prefix))\n",
+    "\n",
+    "    # Embedding\n",
+    "    embedder = TextEmbedder(input_col=\"inputs\",\n",
+    "                            output_col=\"embedding\",\n",
+    "                            engine=\"PyTorch\",\n",
+    "                            model_url=\"djl://ai.djl.huggingface.pytorch/sentence-transformers/all-MiniLM-L6-v2\")\n",
+    "    outputDf = embedder.embed(df)\n",
+    "    outputDf.write.mode(\"overwrite\").parquet(\"s3://\" + os.path.join(args.s3_output_bucket, args.s3_output_key_prefix))\n",
```
Comment (frankfliu) [inline apache-spark/notebook/sagemaker_spark_processing_text.ipynb:122]:
```suggestion
    "    outputDf.write.mode(\"overwrite\").parquet(f\"s3://{args.s3_output_bucket}/{args.s3_output_key_prefix}\")\n",
```

File: apache-spark/notebook/sagemaker_spark_processing_text/sagemaker_spark_processing_text.ipynb [hunk 4]
```diff
@@ -0,0 +1,396 @@
+{
```
Comment (lanking520) [inline apache-spark/notebook/sagemaker_spark_processing_text/sagemaker_spark_processing_text.ipynb:1]: please make this notebook under aws/sagemaker/processing

Comment (xyang16) [reply apache-spark/notebook/sagemaker_spark_processing_text/sagemaker_spark_processing_text.ipynb]: Done

**Review by lanking520 — APPROVED** [review]

**Pull request merged**